### ISSUE TITLE
raise error when I migrate issue in v0.7-rc2

### ISSUE THREAD

**Issue by altblanc** [gotsunami/gitlab-copy#39]
error such as below raised when I migrate issue by v0.7-rc2. this error occured on windows and linux.
```
panic: runtime error: invalid memory address or nil pointer dereference
goroutine 1 [running]:
main.(*migration).migrateIssue
        /home/ab/work/go/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/migration.go:138 main.(*migration).migrate
        /home/ab/work/go/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/migration.go:375 main.main
        /home/ab/work/go/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/main.go:156
```

**Comment by matm** [comment]
Thanks for reporting, we're on it!

**Comment by matm** [comment]
Fixed in df216f1 6 months ago.
